api/proto/teleport/legacy/types/types.proto

```diff
@@ -6536,3 +6536,21 @@ message OktaOptions {
     (gogoproto.casttype) = "Duration"
   ];
 }
+
+// AccessGraphSync is a configuration for Access Graph service.
+message AccessGraphSync {
+  repeated AccessGraphAWSSync AWS = 1 [(gogoproto.jsontag) = "types,omitempty"];
```

Comment:
```suggestion
  repeated AccessGraphAWSSync AWS = 1 [(gogoproto.jsontag) = "aws,omitempty"];
```
It currently looks like this:
```yaml
kind: discovery_config
....
spec:
  aws:
    - types: ["ec2"]
      regions: ["x", "y"]
  access_graph:
    types:
      - account_id: 123456789012
        regions: ["x", "y"]
```

I was expecting to have the following, instead:
```yaml
  access_graph:
    aws:
      - account_id: 123456789012
        regions: ["x", "y"]
```

Reply:
typo :cry: